funnel/models/space.py

```diff
@@ -303,7 +314,21 @@ def all(cls):
         """
         Return currently active events, sorted by date.
         """
-        return cls.query.filter(cls.status >= 1).filter(cls.status <= 4).order_by(cls.date.desc()).all()
+        return cls.query.filter(cls.state.CURRENTLY_LISTED).order_by(cls.date.desc()).all()
+
+    @classmethod
+    def fetch_sorted(cls, profile=None, any_profile=True):
```

Comment:
The `profile` and `any_profile` kwargs don't seem to be used?

Reply:
Oh right. they are not needed. deleting.